### PR TITLE
Feature/task254 test notification

### PR DESCRIPTION
**Pull request by annashamray**
**Changes:**
1. create test for firs 3 steps of notification process:
    - create kanaal (producer)
    - subscribe to kanaal (consumer)
    - send notification triggered by POST-request (producer)
1. update docker- and jenkins-config files with the image of Notification component
1. update versions of docker images
1. fix old test

### CODE REVIEW

**Comment by annashamray** [comment]
I accidentally included commits from `feature/archiefactiedatum` to this branch. 
Can I keep it as it is or should I rebase commits?

**Comment by sergei-maertens** [comment]
@annashamray no problem, I rebased just now :-)

File: tests/test_notificatie_zrc_to_drc.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        drc_auth = ClientAuth(
+            client_id='nc',
+            secret='nc-to-drc',
+            scopes=['notificaties.scopes.publiceren']
+        )
+
+        # can't reat url from spec, because it isn't added to schema
+        # not working because /callbacks endpoint is not included in drc
+        callback_url = '{}callbacks'.format(drc_client.base_url)
```
Comment (sergei-maertens) [inline tests/test_notificatie_zrc_to_drc.py:54]: Feel free to use f-strings for simple interpolation:

```suggestion
        callback_url = f'{drc_client.base_url}callbacks'
```

**Review by sergei-maertens — APPROVED** [review]

**Comment by sergei-maertens** [comment]
test this please

**Pull request merged**